src/frontend/containers/ContentView/LayoutSwitcher.tsx

```diff
@@ -98,8 +99,12 @@ const Layout = ({
         }
         if (e.key === 'ArrowLeft' && index > 0) {
           index -= 1;
+          // TODO: when the activeElement GalleryItem goes out of view, focus will be handed over to the body element:
+          // -> Gallery keyboard shortkeys stop working. So, force focus on container
```

Comment:
@hummingly I couldn't figure out this part, could you take a look?

Reply:
Ok, a really dirty way to be able to receive keyboard events even after losing focus is to set the tabindex of the container to -1 and leave it.

Actually, I'm not sure what you mean. The keyboard navigation in the gallery is more similar to the one in the file tag editor, meaning the active element is the container and the focus is moved by using aria-activedescendant. That means we would need to add tabindex=0 to the gallery div and then remove tabindex from all gallery items. After that manage focus with aria-activedescendent like in the tag selector.

Reply:
Additionally, since you probably want to keep the global arrow navigation, we should check that in all our input handling code we stop propagation of those keys.

Reply:
the situation was:  when you click an image, the activeElement becomes that specific gallery item. When you would scroll down, that element gets removed from the DOM at some point, and the body becomes the activeElement. In this PR I changed some things around so that the gallery container becomes the activeElement, so the shortcuts keys still work most of the time.
But I commented on the wrong TODO, what I could use some help with is the MobX warnings in the Locations panel, there's also a TODO for that

Reply:
So, is that fixed or should I look over that again?

I will take a look at the warnings after dinner.

Reply:
You can take a look if you want, but it's not really needed. It's not completely fixed, but it's better than before: when you do stuff in the toolbar or advanced search, it refocuses the gallery when you're done, which was the main issue. I'm only noticing now that using the left/right cursor keys in the tag popover now closes the tag popover, because the gallery gets focused. We might want to give priority to the input element there. We can leave that for when we do another layer of polish

Reply:
I guess I will first fix the mobx warnings and push the PR. Afterwards, I will take an in depth look at the overall keyboard navigation.